validation/src/main/scala/hmda/validation/engine/ValidationError.scala

```diff
@@ -1,3 +1,12 @@
 package hmda.validation.engine
 
-case class ValidationError(id: String, msg: String)
+sealed trait ValidationErrorType
+case object Syntactical extends ValidationErrorType
+case object Validity extends ValidationErrorType
+case object Quality extends ValidationErrorType
+case object Macro extends ValidationErrorType
+
+case class ValidationError(id: String, msg: String, errorType: ValidationErrorType)
```

Comment:
It's the loan ID for a loan/LAR, and the control number (agency code + respondent ID) for a transmittal sheet.  I agree that renaming would help clarify.